core/tests/wdio.conf.js

```diff
@@ -211,6 +197,43 @@ exports.config = {
    * @param {Object}         browser instance of created browser/device session
    */
   before: function() {
+    // Only running video recorder on Github Actions, since running it on
+    // CicleCI causes RAM issues (meaning very high flakiness).
```

Comment:
CircleCI

Reply:
This comment is not needed as we do not run test on CircleCl now. Removed.